src/containers/shared/components/Transaction/XChainClaim/parser.ts

```diff
@@ -0,0 +1,22 @@
+export function parser(tx: any, meta: any) {
+  const affectedNodes = meta.AffectedNodes
+  const modifiedAccountRoots = affectedNodes.filter(
+    (node: any) =>
+      node.ModifiedNode && node.ModifiedNode.LedgerEntryType === 'AccountRoot',
+  )
+  const doorNode = modifiedAccountRoots.filter(
+    (node: any) =>
+      node.ModifiedNode.FinalFields.Balance <
+      node.ModifiedNode.PreviousFields.Balance,
+  )[0]
```

Comment:
Are there any accounts that could have lost money in a transaction and NOT be the bridge owner? (Maybe this will be answered with the pre-conditions above)

Reply:
Hmm my example is submitted by the door account (aka the bridge owner). Normally, two accounts will lose money: the account that pays the fee and the door account. So I'll have to think about that a little more.

Reply:
Is this resolved now, or not something which needs to be resolved in this PR?